.github/workflows/android_tests.yml

```diff
@@ -42,8 +41,7 @@ jobs:
       - run: ./ci/mac_ci_setup.sh
         name: 'Install dependencies'
       - run: |
-          # /test/kotlin/integration is excluded since it is run on a separate job
-          bazelisk query //test/java/...
+          bazelisk test --test_output=all --build_tests_only  //test/java/...
         name: 'Run Java library tests'
   kotlintestslinux:
```

Comment:
How come ^^ those jobs run in mac instead of linux? Seems like we should run on linux (which will probably get better machines and more concurrency easier).

Also the below target for integration tests is not running the integration test target? https://github.com/envoyproxy/envoy-mobile/blob/b767e175155f9e6135d1b33a82229ed9e0e3c686/.github/workflows/android_tests.yml#L69

Reply:
Talked offline. Related to #1418.

Reply:
The integration target is `//test/kotlin/integration/...` so it's included in the above target. I suggested running them combined (in this and other cases) due to fixed startup overhead and instance queueing.